crates/typst-utils/src/lib.rs

```diff
@@ -6,6 +6,7 @@ pub mod fat;
 mod macros;
 mod bitset;
 mod deferred;
+pub mod format;
```

Comment:
I think this should be moved up, near `fat`

Reply:
Ohh... Yeah, you're right.